spec/requests/as_content_owner/content_owner_spec.rb

```diff
@@ -264,6 +265,32 @@
         it { expect(reference).not_to be }
       end
     end
+
+    describe '.upload_reference_file' do
+      let(:asset) { Yt::Asset.new id: ENV['YT_TEST_PARTNER_ASSET_ID'], auth: $content_owner }
+      let(:match_policy) { Yt::MatchPolicy.new asset_id: ENV['YT_TEST_PARTNER_ASSET_ID'], auth: $content_owner }
+
+      let(:upload_params) { {asset_id: asset.id, content_type: 'video'} }
+      let(:reference) { $content_owner.upload_reference_file path_or_url, upload_params }
+      after { reference.delete }
+
+      before do
+        asset.ownership.update(assetId: asset.id) && asset.ownership.obtain!
```

Comment:
I'm thinking of moving some of this expensive/complicated "fixture" creation to outside the tests themselves.

Reply:
Should we do this in a separate PR or in this one?

Reply:
Oh, this is something I'm planning to do in the future. Just wanted to give you a heads up.